helmfile/values/fluentd-sc.yaml.gotmpl

```diff
@@ -146,6 +146,13 @@ extraConfigMaps:
       </metric>
     </filter>
 
+    <filter **>
+      @type record_transformer
+      <record>
+        cluster.name "{{ .Values.global.environmentName }}"
```

Comment:
Do we want to use `.Values.global.environmentName` or should we create a new entry like `.Values.global.clusterName`?

Reply:
I mean leaning towards a clusterName variable instead. 
Can't from the top of my head think of anywhere elase where we are using environment name than in the deploy scripts,  where it seems to be used to derive a cluster name :D 
https://github.com/elastisys/compliantkubernetes-apps/blob/d73f39d3afb022c804b027cfb9e30bb7b603f84e/scripts/deploy-wc.sh#L14

Reply:
Seem to not be used anywhere else than there 
```bash
$ grep -r CLUSTER_NAME
scripts/deploy-wc.sh:export CLUSTER_NAME="${environment}_${cloud_provider}"
```

So I don't really see the benefit from using a separate variable (if we basically don't use this variable anywhere else). I can see that we want to rename `environmentName` to `clusterName` but I don't see the big benefit of having both

Reply:
Agree, rename `environmentName` to `clusterName`